ament_cmake_target_dependencies/cmake/ament_target_dependencies.cmake

```diff
@@ -78,10 +78,9 @@ function(ament_target_dependencies target)
           if(NOT "${${package_name}_LIBRARY_DIRS}" STREQUAL "")
             if (NOT IS_ABSOLUTE ${library} OR NOT EXISTS ${library})
               find_library(lib NAMES ${library} PATHS ${${package_name}_LIBRARY_DIRS} NO_DEFAULT_PATH)
-              if(NOT lib)
-                message(FATAL_ERROR "ament_target_dependencies() ${library} library not found in ${${package_name}_LIBRARY_DIRS}")
+              if (lib)
```

Comment:
Not space between `if` and `(` to pass linter.

Reply:
Done in https://github.com/ament/ament_cmake/pull/248/commits/5ea24d96eef94214377339043b35ae64bb9d5cee.